syntax = "proto3";

// The greeting service definition.
service MyService {
    // Sends a greeting
    rpc SayHello (HelloRequest) returns (HelloReply) {
    }
}

// The request message containing the question and the texts.
message HelloRequest {
    string question = 1;
    repeated string texts = 2;
}

// The response message containing the best answer extracted from the texts
message HelloReply {
    string answer = 1;
}